src/publishers.rs

```diff
@@ -60,26 +62,61 @@ pub fn publisher_users(
     crate_name: &str,
 ) -> Result<Vec<PublisherData>> {
     let url = format!("https://crates.io/api/v1/crates/{}/owner_user", crate_name);
-    let data: UsersResponse = client.get(&url).call().into_json_deserialize()?;
-    Ok(data.users)
+    if let Some(resp) = retry_get(&url, client, 3) {
+        let data: UsersResponse = resp.into_json_deserialize()?;
+        Ok(data.users)
+    } else {
+        Err(Error::new(
+            ErrorKind::ConnectionReset,
+            "Failed to retrieve publisher users",
+        ))
+    }
 }
 
 pub fn publisher_teams(
     client: &mut RateLimitedClient,
     crate_name: &str,
 ) -> Result<Vec<PublisherData>> {
     let url = format!("https://crates.io/api/v1/crates/{}/owner_team", crate_name);
-    let data: TeamsResponse = client.get(&url).call().into_json_deserialize()?;
-    Ok(data.teams)
+    if let Some(resp) = retry_get(&url, client, 3) {
+        let data: TeamsResponse = resp.into_json_deserialize()?;
+        Ok(data.teams)
+    } else {
+        Err(Error::new(
+            ErrorKind::ConnectionReset,
+            "Failed to retrieve publisher teams",
+        ))
+    }
+}
+
+fn retry_get(url: &String, client: &mut RateLimitedClient, attempts: u8) -> Option<ureq::Response> {
```

Comment:
`&String` is a rather pointless type to use explicitly. This should accept `&str`, to which `&String` automatically dereferences.

Tip: if you run `cargo clippy`, it will point out things like these.

Also, I believe a better name would be `get_with_retries` or some such; `retry_get` sounds like something I'd need to call after `get` failed and I need to retry it.